.ado/pull-request.yml

```diff
@@ -13,7 +13,7 @@ pool:
 
 jobs:
   - job: V8JsiBuild
-    timeoutInMinutes: 180
+    timeoutInMinutes: 210

```

Comment:
>210 [](http://example.com/codeflow?start=22&length=3)

That's like 4 hours, isn't it a bit long?

Reply:
Yes :( But we sometimes get time-outs, especially when building UWP / ARM64 because we need to host-compile tools, and it sucks waiting 3 hours to get the timeout then re-running the slices again. Hopefully this alleviates some of those instances.